src/App.tsx

```diff
@@ -1,61 +1,135 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+const getUser = (id: number): User | null => {
+  return usersFromServer.find(user => user.id === id) || null;
+};
+
+const todoList: Todo[] = todosFromServer.map(todo => (
```

Comment:
Consider using `Array.reduce` instead of `Math.max` to get the maximum id. It is more efficient and readable. 📈